### DISCUSSION TITLE
Ideas to make guardians killable spawnable

### DISCUSSION THREAD

**Discussion by DWAA1660** [commandgeek/geeksmp#61] [Ideas]
So as you may know terra plugin messed up natural mobs spawning in structures so what would be the best way to make guardians killable/spawnable to 1: Make all achievements possible 2: Make it possible to get items you get from there loot.

**Leave any ideas in comments**

**Comment by srnyx** [comment]
We could have an artificial monument (or just find one near spawn) and then add like 4 guardian spawners. Then make it so no one can break/place any blocks at the monument. Would be easy to farm them though.

**Reply by DWAA1660** [reply]
> We could have an artificial monument (or just find one near spawn) and then add like 4 guardian spawners. Then make it so no one can break/place any blocks at the monument. Would be easy to farm them though.

yea i thought about that but there could be really op farm but we could outlaw guardian farms
